src/main/java/org/springframework/cloud/task/config/DefaultTaskConfigurer.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.task.config;
+
+import org.springframework.cloud.task.repository.LoggerTaskRepository;
+import org.springframework.cloud.task.repository.TaskRepository;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+/**
+ * If no TaskConfigurer is present this configuration will be used.
+ * @author Glenn Renfro
+ */
+@Configuration
+public class DefaultTaskConfigurer {
+
+	@Bean
+	@Scope("prototype")
+	public TaskHandler getTaskHandler() {
```

Comment:
Annotating these methods is not correct.  That would require everyone to annotate their methods as well as it prevents us from really overriding this configuration.  What we really want to do is import a configuration that uses the `TaskConfigurer` to provide the bean implementations.  Take a look at `AbstractBatchConfiguration` in batch as an example.